templates/compose/beszel.yaml

```
@@ -6,19 +6,24 @@
-      - /var/run/docker.sock:/var/run/docker.sock:ro
+      - beszel_agent_data:/var/lib/beszel-agent
+      - beszel_socket:/beszel_socket
+      - '/var/run/docker.sock:/var/run/docker.sock:ro'
     environment:
-      - PORT=45876
-      - KEY=${KEY}
+      - LISTEN=/beszel_socket/beszel.sock
+      - HUB_URL=http://beszel:8090
+      - 'TOKEN=${TOKEN}'
+      - 'KEY=${KEY}'
+
```

YAMLlint (1.37.1)

[error] too many blank lines (1 > 0)

(empty-lines)